Cli.py

```diff
@@ -11,7 +11,9 @@
 class Cli:
     def __init__(self):
         pass
-    def session(self):
+    # maybe some things can be added to the constructor
+    def session(self): 
```

Comment:
This is a possibility, but I'd like to hear from you what you think can be `*args`/`**kwargs` in this function :)